opsgenie.yaml

```diff
@@ -84,6 +84,142 @@ systems:
                       call_function: opsgenie.snooze
                       #  alert_Id is exported from the event
 
+      schedules:
+        driver: web
+        rawAction: request
+        parameters:
+          URL: https://api.opsgenie.com/v2/schedules{{ with .ctx.scheduleId }}/{{ . }}?identifierType={{ default "name" $.ctx.scheduleIdType }}{{ end }}
```

Comment:
Are there retries on these? What happens in the case where a single API call fails?